rootfs/usr/local/bin/aws-config-setup

```diff
@@ -59,7 +59,7 @@ crudini --set "${AWS_CONFIG_FILE}" "profile ${AWS_DEFAULT_PROFILE}" source_profi
 read -p "Setup AWS Credentials (aws-vault)? [y/n] " SETUP_VAULT
 
 if [ "${SETUP_VAULT}" == "y" ]; then
-	aws-vault add "${AWS_SOURCE_PROFILE}"
+	aws-vault --backend=file add "${AWS_SOURCE_PROFILE}"
```

Comment:
we have `AWS_VAULT_BACKEND` defined so this should not be needed or at the very least is inconsistent.

Reply:
Recent users seem to have had issues with their `aws-vault` configs being written to the wrong place. Since Geodesic only works with `backend=file` I want to eliminate that failure mode. Because it is where keys get stored, it is particularly painful to repeat if it goes wrong the first time.

Reply:
but this only "fixes" one place in one script. It does not "fix" `assume-role` in `aws-vault.sh` and does not ensure that `aws-vault` run directly will work. Having the same setting set in (2) places is not desirable and breaks functionality with a supported env. Let's reduce the scope of changes.